en/docs/publish/api-microgateway/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,113 @@
+
+Let's get started...
+
+### Step 1 - Setup Microgateway and K8s API Operator in Kubernetes
+
+1.  Download the Microgateway distribution from the 
+    [github release page's](https://github.com/wso2/product-microgateway/releases) Assets and 
+    extract them to a folder of your choice.
+
+    From here onwards, Microgateway distribution extracted 
+    location would be referred as `MG_HOME`.
+
+2.  Using kubectl tool, apply Kubernetes configurations for Microgateway.
+    ```bash
+    kubectl apply -Rf <MG_HOME>/resources/k8s-artifacts
+    ```
+
+3.  Let's create a namespace and install K8s API Operator by executing the following command.
+    ```bash
+    kubectl create ns wso2-system;
+    kubectl apply -n wso2-system -f https://github.com/wso2/k8s-api-operator/releases/download/v2.0.0-m3/api-operator-configs.yaml
+    ```
+
+### Step 2 - Create and deploy an API project
+
+Let's create our first project with name "petstore" by adding the
+[open API definition](https://petstore.swagger.io/v2/swagger.json) of the petstore.
+
+1.  First we need to create a Kubernetes configmap with the swagger definition.
+
+    ```bash
+    kubectl create cm petstore-cm --from-literal=swagger="$(curl -k https://petstore.swagger.io/v2/swagger.json)"
+    ```
+
+    !!! note
+        You can also create an Kubernetes configmap with an zipped APICTL project to deploy an API.
+
+        Download the CLI tool(APICTL) and the microgateway distribution from the 
+        [github release page's](https://github.com/wso2/product-microgateway/releases) Assets and 
+        extract them to a folder of your choice.
+        ```bash
+        export PATH=$PATH:<CLI_TOOL_EXTRACTED_LOCATION>
+        ```
+
+        ```bash
+        apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
+        ```
+
+        The project is now initialized. You should notice a directory with name "petstore" being created in the location 
+        where you executed the command. Let's Zip the created "petstore"` directory and create a Kubernetes condfigmap.
+        You can also use this Kubernetes configmap to deploy APIs.
+
+        ```bash
+        zip -r petstore.zip petstore/
+        kubectl create cm petstore-cm --from-file petstore.zip
+        ```
+
+2.  Now let's deploy our first API by creating an API resource in Kubernetes.
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: wso2.com/v1alpha2
+    kind: API
+    metadata:
+        name: petstore-api
+    spec:
+        swaggerConfigMapName: petstore-cm
+    EOF
+    ```
+
+    Or else execute the following command
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/wso2/k8s-api-operator/v2.0.0-m3/scenarios/scenario-2/petstore-api.yaml
+    ```
+
+### Step 3 - Invoke the API
+
+1.  The next step would be to invoke the API using a REST tool. Since APIs on the Microgateway are by default secured. 
+    We need a valid token in order to invoke the API. 
```

Comment:
How do you get this token?

Reply:
@AmaliMatharaarachchi 

Invoking API part is copied from this doc [1], do we want to direct users to that doc.
But copied the content of it because the host and port should be changed.
[1] https://github.com/wso2/docs-apim/pull/2898/files#diff-9f60fa2620117b1c4ca2cb70033720cb04b7dcf4641741584e72105c0449dc51R79

Reply:
We can give them this test token to invoke the API. Then if he needs to generate an access token, then direct user to token generation doc

Reply:
Added a note to direct the user to generate a token

![image](https://user-images.githubusercontent.com/23183042/105019750-76247280-5a6c-11eb-8a88-da603a53ffa7.png)